g3/services/generate/pr/messages/creator.py

```diff
@@ -51,6 +51,6 @@ def create(self, tone: MessageTone, jira=None, include=None, edit: Optional[int]
 
             title = reviewed_message.split("\n")[1]
             description = reviewed_message.split(title)[1]
-            git.push("origin", git_info.branch, force=True)
+            git.push("origin", git_info.branch)
```

Comment:
`--force` was added so that we can easily override the remote with rebased/amended commits in case of `g pr --edit X` or `g pr` on an already pushed branch with the same name in github.

We had discussed setting this in the configs too but defaulting to `True` would make, again, more sense to me since developers commonly rebase or amend their commits before merging their work to the main branch.

We don't need to disable the force flag to resolve the reported issue, do we?

Reply:
Yeap we don't need to. Personally I would rather be asked if something is going to be forced in remote but no strong opinion.

Reply:
I created [this issue](https://github.com/Workable/g3/issues/117) so that we support that in the context of v1; let's develop them separately.

Reply:
Reverted